lib/providers/commands-service-provider.ts

```diff
@@ -42,11 +42,11 @@ class CommandsServiceProvider implements ICommandsServiceProvider {
 	}
 
 	public getDynamicCommands(): IFuture<string[]> {
-		return this.$screenBuilderService.allSupportedCommands(this.$screenBuilderService.generatorName);
+		return this.$screenBuilderService.allSupportedCommands(this.$injector.resolve("project").getProjectDir().wait(), this.$screenBuilderService.generatorFullName);
```

Comment:
will this generate new projectDir when I write incorrect command (for example `appbuilder add-incorrect`)